custom_components/plugwise_dev/climate.py

```diff
@@ -76,14 +94,19 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
     try:
         api.ping_anna_thermostat()
     except OSError:
-        _LOGGER.debug("Ping failed, retrying later", exc_info=True)
+        _LOGGER.debug(
+            "Ping failed, retrying later", exc_info=True
+        )
         raise PlatformNotReady
     devices = [
         ThermostatDevice(
-            api, config[CONF_NAME], config[CONF_MIN_TEMP], config[CONF_MAX_TEMP]
+            api,
+            config[CONF_NAME],
+            config[CONF_MIN_TEMP],
+            config[CONF_MAX_TEMP],
         )
     ]
-    add_entities(devices, True)
+    async_add_entities(devices)
```

Comment:
If you add the `, True` it will poll on start, not sure if that's the best way to go, but it saves some time for the full async to catch up and update the front-end. (Actually the addition of it in synchronous mode was advised during review of the initial release in HA)

Reply:
I looked at other HA-code, there is hardly ever the additional `, True`. I've seen it once.
But I can test with it.